ml/models/logo_detection/process.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from detect import Detector  # Importing the Detector class from detect.py
+
+def process(input_image_path):
+
+    source = input_image_path
+    detector = Detector(weights='logo_detection.pt', source=source)
```

Comment:
We will need this to be the path to the detection model in the docker build
```suggestion
    detector = Detector(weights='/src/ml/models/logo_detection/logo_detection.pt', source=source)
```
For local testing in isolation to the rest of the API we can update the `main.py` file to use the CWD to find the weights file